CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## [1.57.2](https://github.com/brave/brave-ios/releases/tag/v1.57.2)
+
+- Improved performance of closing tabs for sites where Window.open is called many times. ([#8060](https://github.com/brave/brave-ios/issues/8060))
```

Comment:
hmmm maybe rewording the above so we're not mentioning `Window.open`? Probably doesn't mean much to most users.